index.js

```diff
@@ -37,16 +41,17 @@ const transferFromTogglToTempo = async (from, to, utc, dryRun = false) => {
   // create worklogs in tempo from toggl time entries
   await bluebird.map(
     parsedEntries,
-    async ({ issueKey, start, duration, comment }) => (
+    async ({ issueKey, start, duration, comment }) => {
+      await sleep(Math.random() * 20000)
+      console.log(issueKey + ' ' + comment)
```

Comment:
per linter requirements, template usage: 
```suggestion
      console.log(`${issueKey} ${comment}`)
```